src/vivarium_cluster_tools/psimulate/cluster/cli_options.py

```diff
@@ -7,10 +7,37 @@
 
 """
 
+from typing import List, Optional
+
 import click
 
 _RUNTIME_FORMAT = "hh:mm:ss"
 
+# https://docs.cluster.ihme.washington.edu/#hpc-execution-host-hardware-specifications
+_AVAILABLE_HARDWARE = [
+    "c6320",  # typical
+    "r630",  # high capacity
+    "c6420v1",  # batch 1
+    "c6420v2",  # batch 2
+    "r650",  # high capacity
+    "r650v2",  # high capacity
+    "r650xs",  # high speed
+]
+
+
+def _validate_and_split_hardware(
+    ctx: click.Context, param: click.core.Option, value: Optional[str]
+) -> List[Optional[str]]:
+    hardware = value.split(",") if value else []
+    bad_requests = set(hardware) - set(_AVAILABLE_HARDWARE)
+    if bad_requests:
+        raise click.BadParameter(
```

Comment:
Related to your above GitHub comment, do you think it would be useful to provide the URL in the code comment in the error message? (I see that's in the help already)

Reply:
good idea